test/e2e/apimachinery/table_conversion.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/onsi/gomega"
 
 	authorizationv1 "k8s.io/api/authorization/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Is gofmt adding v1?

Seems like it's trying to distinguish between the two v1s we are importing.

Reply:
> Is gofmt adding v1?

yes

Reply:
I've changed it back

Reply:
I've changed it back.